### DISCUSSION TITLE
Services sociétaux à l'international

### DISCUSSION THREAD

**Discussion by nicolaspkandeel** [incubateur-ademe/nosgestesclimat#1808] [Ideas]
Bonjour,
A ce jour les versions d'autres régions de la France se différencient dans NGC par le FE du mix électrique et des suggestions spécifiques sur le transport.
Il me semble important de régionnaliser, même sommairement, les services sociétaux.
L'un des intérets (de mon point de vue) de l'internationalisation de NGC est de proposer des calculs d'empreinte pour des gens qui ne l'ont pas dans leur pays, un enjeu dans les pays moins développés que l'Europe.
Mais avoir par défaut l'empreinte des services sociétaux français présente un risque de rejet des résultats.
J'ai vu une discussion sur l'intégration des données SDES (https://github.com/datagir/nosgestesclimat/issues/1583), avec une méthode qui pourrait être internationale (quoique limitée à 49zones), mais qui semble compliquée (https://github.com/thomasgibon/io-footprinting/blob/master/miniprojet_mines/1_introduction_structure_IO.ipynb)

Peut-on s'autoriser une évaluation simple à court terme, par exemple sur la base du budget de l'Etat de chaque pays comparé au budget Français, pour faire un ratio ? Afin de ne pas présenter des données à coté de la plaque.
Et ensuite mettre en place des méthodes plus fines, comme par exemple une matrice permettant de convertir les données rapportées au CCNUCC et des données économiques standard en émissions par secteur ?

**Comment by Clemog** [comment]
Hello @nicolaspkandeel !
On a effectivement identifié le sujet : https://github.com/datagir/nosgestesclimat/issues/1715 mais n'avons pas encore pris le temps de travailler dessus

> Peut-on s'autoriser une évaluation simple à court terme, par exemple sur la base du budget de l'Etat de chaque pays comparé au budget Français, pour faire un ratio ? Afin de ne pas présenter des données à coté de la plaque.

Cette proposition me semble être une très bonne idée en V1 effectivement ! 

L'idée par la suite serait que des ambassadeurs/associations/bureaux d'étude travaillent au sein de leur pays pour reproduire https://nosgestesclimat.fr/nouveautés/l'empreinte-climat%20des%20%22services%20sociétaux%22 ([les scripts, le méthode étant ouverte](https://github.com/datagir/nosgestesclimat/wiki/Guide-technique-services-sociétaux) !)

**Comment by Clemog** [comment]
Voir https://github.com/datagir/nosgestesclimat/issues/1715 pour la suite :)
